src/status_im/extensions/core.cljs

```diff
@@ -207,6 +209,28 @@
                            {:failure-event-creator on-failure})
                          {:timeout-ms 5000})}))
 
+(defn- parse-ipfs-add-response [res]
+  (-> (parse-json res)
+      (select-keys [:Name :Hash :Size])
+      (set/rename-keys {:Name :name :Hash :hash :Size :size})))
```

Comment:
It would probably be simpler / more efficient to just destructure key values and reconstruct inline a map wth the right key names.

e.g.
````
(let {{:keys [Name]} (parse-json res)}
  {:name Name})
````

Reply:
👍